doc/classes/Joint3D.xml

```diff
@@ -13,19 +13,21 @@
 		<method name="get_rid" qualifiers="const">
 			<return type="RID" />
 			<description>
-				Returns the joint's [RID].
+				Returns the joint's internal [RID] from the [PhysicsServer3D].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="exclude_nodes_from_collision" type="bool" setter="set_exclude_nodes_from_collision" getter="get_exclude_nodes_from_collision" default="true">
-			If [code]true[/code], the two bodies of the nodes are not able to collide with each other.
+			If [code]true[/code], the two bodies bound together cannot collide with each other.
 		</member>
 		<member name="node_a" type="NodePath" setter="set_node_a" getter="get_node_a" default="NodePath(&quot;&quot;)">
-			The node attached to the first side (A) of the joint.
+			Path to the first node (A) attached to the joint. The node must inherit [PhysicsBody3D].
+			If left empty and [member node_b] is set, the body hangs as if attached to an invisible [StaticBody3D].
```

Comment:
"Hanging" is not correct in all cases, e.g. when the joint has a motor enabled or if there is no gravity, so it should just be "is attached" or "is connected" or something generic like that. Also just "invisible" leaves open what/where this secret body is, so I think it should be e.g. "an invisible fixed [StaticBody3D] without collision shapes".